hadoop-ozone/dist/src/main/smoketest/admincli/scmrole.robot

```diff
@@ -25,4 +25,8 @@ Test Timeout        5 minutes
 *** Test Cases ***
 Run scm roles
     ${output} =         Execute          ozone admin scm roles
-                        Should Match Regexp   ${output}  [scm:9894(:LEADER|)]
\ No newline at end of file
+                        Should Match Regexp   ${output}  [scm:9894(:LEADER|)]
+
+List scm roles as JSON
+    ${output} =         Execute          ozone admin scm roles --json
+                        Should contain   ${output}    "address" :
```

Comment:
We can check that output is valid JSON by passing through `jq`, e.g. find LEADER:

```suggestion
                        Execute    echo '${output}' | jq -r '.[] | select(.raftPeerRole == "LEADER")'
```

Reply:
Thank you, I've incorporated your suggestion in my latest commit